test/screenshot/tab/index.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import Tab, {TabProps} from '../../../packages/tab';
+// https://github.com/material-components/material-components-web-react/issues/513
+// @ts-ignore
+import MaterialIcon from '../../../packages/material-icon/index';
+import '../../../packages/tab-indicator/index.scss';
+import '../../../packages/tab/index.scss';
+import './index.scss';
+
+interface TabContentType {
+  num: number;
+}
+
+const TabContent: (args: TabContentType) => React.ReactElement<any> = ({
+  num, // eslint-disable-line react/prop-types
```

Comment:
It's unclear to me why you are disabling this.  Isn't it just `number`?

Reply:
its eslint-disable...react/prop-types are no needed, and I will address this in a later issue

Reply:
https://github.com/material-components/material-components-web-react/issues/555